### PR TITLE
[ch15602] Remove hreflang for US-based groups

### PR DESCRIPTION
**Pull request by meetup-fkukuev**
Related ticket: https://app.clubhouse.io/meetup/story/15602/remove-hreflang-tags-add-noindex-for-all-us-groups-and-events

Tested locally with 25.2.3299-beta build

### CODE REVIEW

File: packages/mwp-app-render/src/util/seo/links.js [hunk 1]
```diff
@@ -28,14 +28,16 @@ export const generateCanonicalUrl = (
...
  * @param  {String} route   redux'd route to the current page
  * @return {Array}  array of React.element's
  */
 export const generateCanonicalUrlLinkTags = (
 	baseUrl,
 	localeCode,
 	route,
-	forcedLocaleCode = ''
+	forcedLocaleCode = '',
+	groupCountry
```
Comment (andeliv) [inline packages/mwp-app-render/src/util/seo/links.js:40]: web-platform shouldn't be aware of groups or any data from consumer Apps, Let's rename this to smth like `isGenerateAlternateLinks`

File: packages/mwp-app-render/src/util/seo/links.test.js [hunk 2]
```diff
@@ -4,12 +4,14 @@ describe('generateCanonicalUrlLinkTags', () => {
 	const MOCK_BASE_URL = 'http://www.mock-base-url.com';
 	const MOCK_ROUTE = '/mockroute';
 	const MOCK_LOCALE_CODE = 'fr-FR';
+	const MOCK_IS_GENERATE_ALTERNATE_LINKS = true;
```
Comment (andeliv) [inline packages/mwp-app-render/src/util/seo/links.test.js:7]: Please add test for false case

File: packages/mwp-app-render/src/util/seo/links.js (hunk 1 above)
Comment (meetup-fkukuev) [reply packages/mwp-app-render/src/util/seo/links.js]: fixed, updated groupCountry to isGenerateAlternateLinks

File: packages/mwp-app-render/src/util/seo/links.test.js (hunk 2 above)
Comment (meetup-fkukuev) [reply packages/mwp-app-render/src/util/seo/links.test.js:7]: Added the test, thanks!

**Review by andeliv — APPROVED** [review]

File: packages/mwp-app-render/src/util/seo/links.js [hunk 3]
```diff
@@ -28,13 +28,15 @@ export const generateCanonicalUrl = (
...
  * @param  {String} forcedLocaleCode  locale of group
+ * @param {Boolean} isGenerateAlternateLinks Defines if we need to generate alternate link tags for the page
  * @param  {String} route   redux'd route to the current page
  * @return {Array}  array of React.element's
  */
 export const generateCanonicalUrlLinkTags = (
 	baseUrl,
 	localeCode,
 	route,
+	isGenerateAlternateLinks,
```
[the remark this answers is not in the document]
Comment (meetup-fkukuev) [reply packages/mwp-app-render/src/util/seo/links.js:39]: Added isGenerateAlternateLinks to defaultProps

**Pull request merged**